client/signup/previous-step-button/index.jsx

```diff
@@ -30,27 +28,31 @@ export default React.createClass( {
 		const previousStepName = signupUtils.getPreviousStepName( this.props.flowName, this.props.stepName ),
 			{ stepSectionName } = find( this.props.signupProgress, { stepName: previousStepName } );
 
-		return signupUtils.getStepUrl( this.props.flowName, previousStepName, stepSectionName, i18n.getLocaleSlug() );
-	},
+		return signupUtils.getStepUrl( this.props.flowName, previousStepName, stepSectionName, getLocaleSlug() );
+	}
 
-	recordClick() {
+	recordClick = () => {
 		analytics.tracks.recordEvent( 'calypso_signup_previous_step_button_click', {
 			flow: this.props.flowName,
-			step: this.props.stepName
+			step: this.props.stepName,
 		} );
-	},
+	}
 
 	render() {
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
This will lead to a lint error, but let's remove this, since we do want to update this in the future.